.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Was it committed in by mistake?

We keep configuration in `package.json` and it's important we test against syntax as supported in Node.js v6

So that file definitely should not be part of a PR :)

Reply:
I will move this into the samples directory as this was created to support `async/await` syntax in the sample code